swig/python/test_comedi_command.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python
```

Comment:
Why renaming the file to `test_comedi_command.py` in this commit, and renaming it back to 'test_comedi.py` in the next commit?